rstar/src/object.rs

```diff
@@ -145,7 +145,12 @@ pub trait RTreeObject {
 /// assert!(circle.contains_point(&[1.0, 0.0]));
 /// ```
 pub trait PointDistance: RTreeObject {
-    /// Returns the squared euclidean distance between an object to a point.
+    /// Returns the squared distance between an object to a point.
```

Comment:
```suggestion
    /// Returns the squared distance between an object and a point.
```

Or alternatively: 

```
 /// Returns the squared distance from an object to a point.
```

(I know this wasn't your text originally 😉)